ncm-spma/src/main/pan/components/spma/config.pan

```diff
@@ -8,7 +8,7 @@ unique template components/${project.artifactId}/config;
 variable CONFIG_MODULES_CONFIG_SUFFIX ?= 'rpm';
 
 include { 'components/${project.artifactId}/schema' };
-include { 'components/${project.artifactId}/functions' };
+include { 'components/${project.artifactId}/functions-'+PACKAGER };
```

Comment:
can we create subdirs per flavour? e.g. 
```pan
include { format('components/${project.artifactId}/%s/functions', PACKAGER) };
```
(maybe keep a default around for backwards compatibility)?

or make a `components/spma/ips/functions.pan` which is just a `include components/spma/functions`?